source/extensions/transport_sockets/tls/context_manager_impl.cc

```diff
@@ -18,7 +18,8 @@ namespace Tls {
 ContextManagerImpl::ContextManagerImpl(TimeSource& time_source) : time_source_(time_source) {}
 
 ContextManagerImpl::~ContextManagerImpl() {
-  KNOWN_ISSUE_ASSERT(contexts_.empty(), "https://github.com/envoyproxy/envoy/issues/10030");
+  // KNOWN_ISSUE_ASSERT(contexts_.empty(), "https://github.com/envoyproxy/envoy/issues/10030");
```

Comment:
clang-tidy is complaining that since this is trivial you should use `~ContextManagerImpl() = delete` instead. I think you can do that and delete the comment here and the commented out assert, since the new test would trigger this assert.

Reply:
Done. Fixed CI issues, and add one more line code to make sure the test cover the updated code path.